### DISCUSSION TITLE
Does terminal support mouse input similar to how VSCode does?

### DISCUSSION THREAD

**Discussion by GaryAtlan82** [microsoft/terminal#15733] [Q&A] [answered]
Allot of times, I already have my hand on the mouse and just need to make a quick selection or move the cursor around **using** the mouse.

To do standard things such as place the cursor at a certain word and highlight one or more words. No different than in this example in Code:

![](https://i.imgur.com/2vAzmXW.gif)

To be clear, I am not talking about selecting strings that have out have been output by a previous command but the command currently being typed by, for example, PowerShell. 

I did allot of searching and could not come to a conclusion. Some sources say that such a feature is only available in a WSL Windows Terminal, while others say its available in eveywhere on Windows Terminal but needs to be enabled first. 

I dont use WSL at all but PowerShell.

Is there a way to do this in standard windows Terminal or atleast where PowerShell and CMD are concerned?

Cheers!

**Comment by zadjii-msft** [comment, marked as answer]
This is not something the Terminal currently supports, and supporting it is a **H**ard problem. #8573 has a LONG discussion on the technical challenges here. We can discuss more there. Sound good?

**Reply by GaryAtlan82** [reply]
Yh, thats good. I will check your link. Thank you.
